start-github-runner-manager.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set +eu
+
+if [ $# -lt 3 ]
+then
+  echo "usage: $0 REPO TOKEN RUNNER_LABEL "
+  exit 1
+fi
+
+# Build the manager image
+echo "Building github runner manager image..."
+(
+  cd images/github-runners-manager || exit 1
+  ./build.sh
+) || exit 1
+
+# Build the github runner image
+echo "Building github runner image..."
+(
+  cd images/github-runner || exit 1
+  ./build.sh
+) || exit 1
+
+MODEL_FOLDERS="/mnt/models"
+if [ ! -d "$MODEL_FOLDERS" ]; then
+  mkdir -p $MODEL_FOLDERS
+fi
+
+if [ -z "$DOWNLOAD_MODELS" ] || [ "$DOWNLOAD_MODELS" == "ON" ] ; then
+  (
+    set +eu
+    (
+      echo "Building models downloader..."
+      cd images/llama.cpp-model-downloader || exit 1
+      ./build.sh
+    ) || exit 1
+
+    # Kill any dangling container
+    (docker ps    | grep -q llama.cpp-model-downloader) && docker kill llama.cpp-model-downloader && sleep 5
+    (docker ps -a | grep -q llama.cpp-model-downloader) && docker rm llama.cpp-model-downloader   && sleep 5
+
+    echo "Downloading models..."
+    MODELS="ggml-org/models:phi-2/ggml-model-q4_0.gguf ggml-org/models:phi-2/ggml-model-q8_0.gguf"
+    for MODEL in $MODELS
+    do
+      IFS=':'; S=($MODEL); unset IFS;
+      HF_REPO="${S[0]}"
+      HF_FILE="${S[1]}"
+      echo "    HF_REPO=$HF_REPO HF_FILE=$HF_FILE"
+      docker run \
+            --rm \
+            -it \
+            --name llama.cpp-model-downloader \
+            --gpus all \
+            -t \
```

Comment:
```suggestion
            -t \
            -u "1000:1000"
```

Without this, the downloaded file will be owned by root, which will **maybe** inaccessible from runner container (which uses `1000:1000`)

Reply:
yes but we are in rootless mode, but agree need to be fixed